docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>This is fine, but I think you can do this even faster by using <code>ax = az.plot_trace(trace, var_names=..., lines=...)</code> and then passing <code>ax=ax</code> to the second call to <code>az.plot_trace</code>. See <a href="https://arviz-devs.github.io/arviz/api/generated/arviz.plot_trace.html" target="_blank">here</a> and <a href="https://docs.pymc.io/notebooks/multilevel_modeling.html" target="_blank">here</a> for examples ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>